railties/test/generators/scaffold_controller_generator_test.rb

```diff
@@ -221,14 +221,11 @@ def test_model_name_option
     end
 
     assert_file "app/views/admin/users/index.html.erb" do |content|
-      assert_match("'Show', [:admin, user]", content)
-      assert_match("'Edit', edit_admin_user_path(user)", content)
-      assert_match("'Destroy', [:admin, user]", content)
-      assert_match("'New User', new_admin_user_path", content)
+      assert_match("'New user', new_admin_user_path", content)
     end
 
     assert_file "app/views/admin/users/new.html.erb" do |content|
-      assert_match("'Back', admin_users_path", content)
+      assert_match("\"Back to user\", admin_users_path", content)
```

Comment:
```suggestion
      assert_match("\"Back to users\", admin_users_path", content)
```

the s was missing, in users which fixes this test.